assets/js/index.js

```diff
@@ -0,0 +1,67 @@
+window.onload = () => {
+  // CHECK PHISHING-WARNING POPUP'S STATUS
+  let phishingPopUp = "Enabled";
```

Comment:
You could try to give phishingPopUp a value of true or false. this way the if's will be simpler to read.
You could rename the variable to hasPhishingPopUp

Reply:
Its fixed now.

Reply:
Can you review it again now?